packages/reports/src/Containers/profit-table.tsx

```diff
@@ -1,31 +1,59 @@
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
-import { PropTypes as MobxPropTypes } from 'mobx-react';
 import React from 'react';
 import { withRouter } from 'react-router';
 import { DesktopWrapper, MobileWrapper, DataList, DataTable } from '@deriv/components';
-import { extractInfoFromShortcode, isForwardStarting, getUnsupportedContracts, getContractPath } from '@deriv/shared';
+import {
+    extractInfoFromShortcode,
+    isForwardStarting,
+    getContractPath,
+    getSupportedContracts,
```

Comment:
nice :+1: 
please change the import path for this utility in other places of `reports` package
for example: `packages/reports/src/Containers/statement.tsx `

Reply:
please check the other utilities of `_common/contract` file and if the utility is available in the shared package, remove it from `_common/contract`